asv.conf.json

```diff
@@ -24,6 +24,7 @@
   // "install_command": ["in-dir={env_dir} python -mpip install {wheel_file}"],
   // "uninstall_command": ["return-code=any python -mpip uninstall -y {project}"],
   "build_command": [
+    "/bin/rm -rf pybind11",
```

Comment:
`asv` errors out for some benchmarks as it clones `pybind11` multiple times (for each `class` or each python file? not very sure). Removing the existing directory before cloning works.